library/gui/screen/src/main/java/org/quiltmc/qsl/screen/api/client/ScreenEvents.java

```diff
@@ -139,13 +137,13 @@ public final class ScreenEvents {
 	@ClientOnly
 	@FunctionalInterface
 	public interface BeforeInit extends ClientEventAwareListener {
-		void beforeInit(Screen screen, MinecraftClient client, int scaledWidth, int scaledHeight);
+		void beforeInit(Screen screen, MinecraftClient client, boolean firstInit);
```

Comment:
I would like to see some documentation on what `firstInit` is

Reply:
I'll write some documentation for the events